var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -59,6 +59,11 @@ class Gromacs(CMakePackage):
                     'IBM_QPX', 'Sparc64_HPC_ACE', 'IBM_VMX', 'IBM_VSX',
                     'ARM_NEON', 'ARM_NEON_ASIMD'))
     variant('rdtscp', default=True, description='Enable RDTSCP instruction usage')
+    variant('mdrun_only', default=False,
+            description='Enables the build of a cut-down version' +
+                         'of libgromacs and/or the mdrun program')
```

Comment:
You need a space here, otherwise the description will be missing a space when you run `spack info`.

Reply:
Thank you for your help!
I see what you mean!
I added a space and checked the description.